tests/common/utils.ts

```diff
@@ -0,0 +1,180 @@
+import {
+    zkVerifySession,
+    ProofType,
+    TransactionInfo,
+    TransactionStatus,
+    VerifyTransactionInfo,
+    VKRegistrationTransactionInfo
+} from '../../src';
+import {
+    handleCommonEvents,
+    handleEventsWithAttestation,
+    EventResults,
+} from './eventHandlers';
+import path from "path";
+import fs from "fs";
+
+
+export interface ProofData {
+    proof: any;
+    publicSignals: any;
+    vk?: string;
+}
+
+export const loadProofData = (proofType: ProofType): ProofData => {
+    const dataPath = path.join(__dirname, 'data', `${proofType}.json`);
+    return JSON.parse(fs.readFileSync(dataPath, 'utf8'));
+};
+
+export const loadVerificationKey = (proofType: ProofType): string => {
+    if (proofType === ProofType.ultraplonk) {
+        const vkPath = path.join(__dirname, 'data', 'ultraplonk_vk.bin');
+        return fs.readFileSync(vkPath).toString('hex');
+    } else {
+        const dataPath = path.join(__dirname, 'data', `${proofType}.json`);
+        const proofData: ProofData = JSON.parse(fs.readFileSync(dataPath, 'utf8'));
+        return proofData.vk!;
+    }
+};
```

Comment:
Mmm perhaps here I would keep it more generic: no assumptions on the format and file names. After merging https://github.com/HorizenLabs/ultraplonk_verifier/pull/7 we can probably deprecate Noir <= v0.3.0 and things will be easier. But let's keep it for the future, don't want to disrupt anything before hackatons

Reply:
These are just tests, the format doesn't really matter for the files, the tests will turn the data into the zkverifyjs packages required format

Reply:
Got it. But since they might be used by developers as guidance, let's make sure to add a test flow later when supporting Noir >= 0.31.0